test/e2e/smoke-test/packages/contour/1.18.2/contour-values.yaml

```diff
@@ -0,0 +1,4 @@
+---
+envoy:
```

Comment:
The indentation in this file looks a bit off. We can reduce the number of spaces in the indentation